src/Appwrite/Auth/OAuth2.php

```diff
@@ -109,6 +116,38 @@ protected function getScopes():array
         return $this->scopes;
     }
 
+    /**
+     * @param string $code
+     *
+     * @return string
+     */
+    public function getAccessToken(string $code):string
+    {
+        $tokens = $this->getTokens($code);
+        return $tokens['access_token'];
+    }
+
+    /**
+     * @param string $code
+     *
+     * @return string
+     */
+    public function getRefreshToken(string $code):string
+    {
+        $tokens = $this->getTokens($code);
+        return $tokens['refresh_token'];
```

Comment:
We could use null safety here to make the code a bit more solid in case the response hasn't came in as expected.

Reply:
Added null safety